### DISCUSSION TITLE
Offering to Contribute to Firefly Documentation

### DISCUSSION THREAD

**Discussion by poupouproject** [firefly-iii/firefly-iii#8445] [Firefly III] [answered]
Hi there,

I'm a new user of Firefly, and I've fallen in love with the software. I've tried multiple installations, including Docker on Ubuntu, Self-Managed, and the Home Assistant add-on.

By day, I work as a functional analyst and I'd like to contribute to your project in my free time during the evenings. I'm particularly interested in enhancing the documentation, such as adding screenshots and demos.

Is there a specific repository for documentation where I can propose changes or add new information? If so, could you guide me on how to submit a pull request for reviewing and approving these changes?

Thank you for creating such an amazing tool, and I look forward to potentially contributing to its development.

**Comment by JC5** [comment, marked as answer]
Your help is always welcome! Please see "Contributing text and documentation" on this page: https://docs.firefly-iii.org/references/support/

Every page in the documentation has a little edit button in the top right corner. When you press it, you end up in the documentation repository: https://github.com/firefly-iii/docs

**Reply by poupouproject** [reply]
Thank you for your prompt response! I will familiarize myself further with the app and then start updating some parts. Would you be open to me adding a section on how to install your software with Home Assistant?

**Reply by JC5** [reply]
Sure, somewhere in the how-to section, I think. Let me know if I can help.
